src/core/Akka.Tests/Routing/ConsistentHashingRouterSpec.cs

```diff
@@ -231,6 +231,7 @@ public void Consistent_hashing_router_must_adjust_node_ring_when_routee_dies()
             var router5 =
                 Sys.ActorOf(Props.Create<Echo>().WithRouter(new ConsistentHashingPool(2, null, null, null)), "router5");
 
+            AwaitCondition(() => ((RepointableActorRef)router5).IsStarted); //verify that the underlying cell has started
```

Comment:
Added a check here that guarantees that the underlying `Router` has started before we perform the assertion on it.